Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM fedora:25 as builder
+
+RUN dnf install -y \
+    git-core \
+    gcc \
+    gcc-c++ \
+    boost-devel \
+    cmake \
+    autoconf \
+    automake \
+    libtool \
+    mariadb-devel \
+    file \
+    hostname \
+    mariadb-server
+
+RUN git clone --branch master --single-branch --depth 1 https://github.com/anton-povarov/meow /_src/meow
+RUN curl -L https://github.com/nanomsg/nanomsg/archive/1.1.5.tar.gz | tar xvz -C /tmp && mv -v /tmp/nanomsg-1.1.5 /_src/nanomsg
+
+COPY . /_src/pinba2
+RUN /_src/pinba2/docker/build-from-source.sh
+
+FROM fedora:25
+MAINTAINER Anton Povarov "anton.povarov@gmail.com"
+
+RUN dnf install -y file hostname mariadb-server
```

Comment:
@anton-povarov / @mkevac maybe it will be better to use official docker image as a base locked on a particular version rather than install latest available `mariadb-server` during build runtime, WDYT?

* image https://hub.docker.com/_/mariadb/